enterprise/server/cmd/ci_runner/main.go

```diff
@@ -154,6 +147,15 @@ var (
 	gitFetchFilters    = flag.Slice("git_fetch_filters", []string{}, "Filters to apply to `git fetch` commands.")
 	gitFetchDepth      = flag.Int("git_fetch_depth", 0, "Depth to use for `git fetch` commands.")
 
+	// Flags to configure setting up git repo
```

Comment:
nit: should the `git_*` flags and `patch_uri` flag also be moved to this section? since those also affect the git repo state